arches/app/templates/views/components/datatypes/resource-instance.htm

```diff
@@ -35,13 +35,19 @@
                             <i class="fa fa-trash"></i>
                         </button>
                     </div>
-                    <div class="rr-table-column" tabindex="0" role="button" style="flex-grow: 1; padding-left: 10px;"
-                        data-bind="
+                    <div class='rr-table-column icon-column'>
+                        <button 
+                            data-bind="
                             click: function() { if($parent.isEditable !== false){self.toggleSelectedResource($data)}; },
                             clickBubble: false,
                             onEnterkeyClick, onSpaceClick," 
```

Comment:
With this change, the space action no longer works; it just flickers:


https://github.com/archesproject/arches/assets/38668450/da170fad-a622-4867-983e-fd648fa3e9d7

Reply:
Still seeing this in the latest, did you have a look?

I'm showing this is fixed with
```diff
diff --git a/arches/app/templates/views/components/datatypes/resource-instance.htm b/arches/app/templates/views/components/datatypes/resource-instance.htm
index 60d6eb9f7..eceb1ff07 100644
--- a/arches/app/templates/views/components/datatypes/resource-instance.htm
+++ b/arches/app/templates/views/components/datatypes/resource-instance.htm
@@ -41,8 +41,7 @@
                             data-bind="
                             click: function() { if($parent.isEditable !== false){self.toggleSelectedResource($data)}; },
                             attr: {'aria-label': $root.translations.configureRelatedResourceRelationship, 'disabled': $parent.isEditable === false,},
-                            clickBubble: false,
-                            onEnterkeyClick, onSpaceClick," 
+                            clickBubble: false,"
                             style="cursor: pointer;">
                             <i class="fa fa-gear"></i>
                         </button>
```

Reply:
fix in latest